datafusion/sql/src/planner.rs

```diff
@@ -230,6 +229,41 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         Ok(Schema::new(fields))
     }
 
+    pub(super) fn build_column_defaults(
+        &self,
+        columns: &Vec<SQLColumnDef>,
+        planner_context: &mut PlannerContext,
+    ) -> Result<Vec<(String, Expr)>> {
+        let mut column_defaults = vec![];
+        // Default expressions are restricted, column references are not allowed
+        let empty_schema = DFSchema::empty();
+        let human_readable_error = |e: DataFusionError| match e {
+            DataFusionError::SchemaError(SchemaError::FieldNotFound { .. }) => {
+                DataFusionError::Plan(format!(
```

Comment:
please use `plan_err!` macro

Reply:
I’ve tried using it once, but it returned a `Result`, what is needed here is an `Error`.

Reply:
> I’ve tried using it once, but it returned a `Result`, what is needed here is an `Error`.

You are right, in this specific case `plan_datafusion_err!` will work

Reply:
I have changed it to `plan_datafusion_err`. 
Thank you @comphead .The code looks cleaner now.